packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/MarkersController.java

```diff
@@ -88,15 +111,22 @@ void isInfoWindowShown(String markerId, MethodChannel.Result result) {
     if (markerController != null) {
       result.success(markerController.isInfoWindowShown());
     } else {
-      result.error("Invalid markerId", "isInfoWindowShown called with invalid markerId", null);
+      result.error(
```

Comment:
Reverted the change, as just noticed that this changes the existing behaviour and is breaking change.
Eventhough it would be nice to give extra information for the developers that the error could be caused because marker is removed by the clusterManager, changing the existing API might not be an option here.